src/responseApi.js

```diff
@@ -0,0 +1,9 @@
+function getResponse(responseCity) {
+    return fetch(
+      `http://api.weatherstack.com/current?access_key=6ef04114106f0d26e10965a03e92a579&query=${responseCity}`
+    )
+      .then((response) => response.json())
+      .then((response) => response);
+  }
```

Comment:
```suggestion
const getResponse = (responseCity) => 
    fetch(
      `http://api.weatherstack.com/current?access_key=6ef04114106f0d26e10965a03e92a579&query=${responseCity}`
    )
      .then((response) => response.json())
      .then((response) => response);
```
по желанию можно переписать как стрелочную функцию